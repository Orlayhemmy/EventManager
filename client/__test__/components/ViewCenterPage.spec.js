// import React from 'react';
// import { ViewCenterDetails } from '../../components/CenterDetails/Container/viewCenterPage';
// import { props, logout, location } from './defaultProps';

// describe('AddCenter Page component', () => {
//   const center = {
//     message: '',
//     status: 202
//   };
//   const center2 = {
//     message: '',
//     status: 498
//   };
//   const center3 = {
//     message: '',
//     status: 401
//   };
//   it('should render without throwing error', () => {
//     const wrapper = shallow(<ViewCenterDetails
//         auth={props.userState}
//         center={center}
//         logout={logout}
//         location={props.location}
//       />);
//     expect(wrapper).toMatchSnapshot();
//   });
//   it('should get the length of div', () => {
//     const wrapper = shallow(<ViewCenterDetails
//       auth={props.userState}
//         center={center2}
//         logout={logout}
//         location={props.location}
//     />);
//     const centerdiv = wrapper.find('div');
//     expect(centerdiv.length).toEqual(1);
//   });
//   it('should get the length of div', () => {
//     const wrapper = shallow(<ViewCenterDetails
//       auth={props.userState}
//         center={center3}
//         logout={logout}
//         location={props.location}
//     />);
//     const centerdiv = wrapper.find('div');
//     expect(centerdiv.length).toEqual(1);
//   });
// });
