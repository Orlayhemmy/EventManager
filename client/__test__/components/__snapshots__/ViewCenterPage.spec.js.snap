// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddCenter Page component should render without throwing error 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ViewCenterDetails
    auth={
      Object {
        "isAuth": true,
        "user": Object {
          "isAdmin": true,
        },
      }
    }
    center={
      Object {
        "message": "",
        "status": 202,
      }
    }
    location={
      Object {
        "pathname": "/",
      }
    }
    logout={[MockFunction]}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Connect(NavBar)
          path="/"
        />,
        <Connect(CenterDetailsContent)
          path="/"
        />,
        <Footer />,
      ],
      "className": "page-wrapper",
      "id": "view-center-wrapper",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "path": "/",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "path": "/",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {},
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Connect(NavBar)
            path="/"
          />,
          <Connect(CenterDetailsContent)
            path="/"
          />,
          <Footer />,
        ],
        "className": "page-wrapper",
        "id": "view-center-wrapper",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "path": "/",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "path": "/",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {},
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
